includes/embeds/class-amp-soundcloud-embed.php

```diff
@@ -48,7 +48,6 @@ public function unregister_embed() {
 	 */
 	public function oembed( $matches, $attr, $url ) {
 		_deprecated_function( __METHOD__, '0.6' );
-		unset( $matches, $attr );
```

Comment:
This is here on purpose to suppress static analysis warnings about unused function args.

Reply:
Without that I got warnings about unneeded `unset()`s, so... 🤷‍♂ 

I guess ideally we'd ignore warnings about unused function args for hook callbacks.